### PR TITLE
Add conversion algorithm and update endpoint

### PR DESCRIPTION
**Pull request by miguelalba**
This PR includes a metadata conversion algorithm and updates the endpoint in FskService. This commit seems larger than it is. Most of the changes (1900 lines) are from the example JSON metadata for test and a copy of the swagger schema. These two files are needed resources.

The actual changes to check are the algorithm (**ConversionUtils**, ~150 lines) and the **service** (~130). The algorithm is added a small test from a generic model to a process model. More tests may be added for other classes if we find annotated models. Regarding the implementation I rewrote it to use the library snakeyaml instead of the Swagger parser since the latter has many implicit dependencies that need to be added to the osgi projects manually. Snakeyaml instead is a single file.

This can be tested with (after setting port to 9999):
```java
public class ConversionTest {

	public static void main(String[] args) throws IOException {

		ObjectMapper mapper = new ObjectMapper();

		URL url = new URL("http://localhost:9999/convertMetadata/processModel");
		HttpURLConnection conn = (HttpURLConnection) url.openConnection();
		conn.setRequestMethod("POST");
		conn.setRequestProperty("Content-Type", "application/json; utf-8");
		conn.setRequestProperty("Accept", "application/json");
		conn.setDoOutput(true);

		GenericModel exampleModel = new GenericModel()
				.generalInformation(new GenericModelGeneralInformation().name("Toy model"));
		exampleModel.setModelType("genericModel");
		try (OutputStream outputStream = conn.getOutputStream()) {
			mapper.writeValue(outputStream, exampleModel);
		}

		// Response
		int status = conn.getResponseCode();

		conn.disconnect();
	}
}
```

### CODE REVIEW

File: de.bund.bfr.knime.fsklab.service/src/de/bund/bfr/knime/fsklab/service/FskService.java [hunk 1]
```diff
@@ -285,110 +295,6 @@ public String render(Object model) {
...
-            .generalInformation(new PredictiveModelGeneralInformation()
-                .modelCategory(new ModelCategory().modelClass("Toxicological reference value")))
-            .scope(new ToxicologicalModelScope())
-            .dataBackground(new PredictiveModelDataBackground())
-            .modelMath(new GenericModelModelMath())
-            .modelType("toxicologicalModel");
-      default:
-        return null;
-    }
-  }
```
Comment (schuelet) [inline de.bund.bfr.knime.fsklab.service/src/de/bund/bfr/knime/fsklab/service/FskService.java:391]: remove unused import packages: 
import de.bund.bfr.metadata.swagger.ConsumptionModel;
import de.bund.bfr.metadata.swagger.ConsumptionModelScope;
import de.bund.bfr.metadata.swagger.DataModel;
import de.bund.bfr.metadata.swagger.DataModelGeneralInformation;
import de.bund.bfr.metadata.swagger.DataModelModelMath;
import de.bund.bfr.metadata.swagger.DoseResponseModel;
import de.bund.bfr.metadata.swagger.DoseResponseModelGeneralInformation;
import de.bund.bfr.metadata.swagger.DoseResponseModelModelMath;
import de.bund.bfr.metadata.swagger.DoseResponseModelScope;
import de.bund.bfr.metadata.swagger.ExposureModel;
import de.bund.bfr.metadata.swagger.ExposureModelScope;
import de.bund.bfr.metadata.swagger.GenericModel;
import de.bund.bfr.metadata.swagger.GenericModelDataBackground;
import de.bund.bfr.metadata.swagger.GenericModelGeneralInformation;
import de.bund.bfr.metadata.swagger.GenericModelModelMath;
import de.bund.bfr.metadata.swagger.GenericModelScope;
import de.bund.bfr.metadata.swagger.HealthModel;
import de.bund.bfr.metadata.swagger.HealthModelScope;
import de.bund.bfr.metadata.swagger.Model;
import de.bund.bfr.metadata.swagger.ModelCategory;
import de.bund.bfr.metadata.swagger.OtherModel;
import de.bund.bfr.metadata.swagger.OtherModelDataBackground;
import de.bund.bfr.metadata.swagger.OtherModelGeneralInformation;
import de.bund.bfr.metadata.swagger.OtherModelModelMath;
import de.bund.bfr.metadata.swagger.OtherModelScope;
import de.bund.bfr.metadata.swagger.PredictiveModel;
import de.bund.bfr.metadata.swagger.PredictiveModelDataBackground;
import de.bund.bfr.metadata.swagger.PredictiveModelGeneralInformation;
import de.bund.bfr.metadata.swagger.PredictiveModelModelMath;
import de.bund.bfr.metadata.swagger.PredictiveModelScope;
import de.bund.bfr.metadata.swagger.ProcessModel;
import de.bund.bfr.metadata.swagger.ProcessModelScope;
import de.bund.bfr.metadata.swagger.QraModel;
import de.bund.bfr.metadata.swagger.RiskModel;
import de.bund.bfr.metadata.swagger.ToxicologicalModel;
import de.bund.bfr.metadata.swagger.ToxicologicalModelScope;

File: de.bund.bfr.knime.fsklab.service/src/de/bund/bfr/knime/fsklab/service/FskService.java [hunk 2]
```diff
@@ -172,13 +177,18 @@ public void run() {
 
     // input metadata as body parameter
     get("convertMetadata/:targetModelClass", (req, res) -> {
```
Comment (schuelet) [inline de.bund.bfr.knime.fsklab.service/src/de/bund/bfr/knime/fsklab/service/FskService.java:179]: should that not be a POST method?

**Review by schuelet — CHANGES_REQUESTED** [review]
Cant access the endpoint, so I cant validate if the algorithm works.

**Comment by miguelalba** [comment]
Ok. I fixed the conflicts and provided a test. This can be tested and merged to development.

**Review by schuelet — APPROVED** [review]

**Pull request merged**